sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/blob/perf/ListBlobsTest.java

```diff
@@ -25,12 +24,15 @@ public Mono<Void> globalSetupAsync() {
         // drastically less CPU usage and throughput, there is ongoing discussions with Reactor Netty on what causes
         // this edge case, whether we had a design flaw in the performance tests, or if there is a configuration change
         // needed in Reactor Netty.
+        int parallel = options.getParallel();
         return super.globalSetupAsync().then(
             Flux.range(0, options.getCount())
-                .parallel(options.getParallel())
-                .runOn(Schedulers.boundedElastic())
+                .parallel(parallel)
+                .runOn(Schedulers.parallel())
                 .flatMap(iteration -> blobContainerAsyncClient.getBlobAsyncClient("getblobstest-" + UUID.randomUUID())
-                    .upload(Flux.empty(), null), false, Math.min(options.getParallel(), 1000 / options.getParallel()), 1)
+                    .getBlockBlobAsyncClient()
+                    .upload(Flux.empty(), 0L), false, Math.min(1000 / parallel, parallel), 1)
```

Comment:
Is this instance of `Math.min(1000 / parallel, parallel)` still needed?

Reply:
No, the Math.min isn't necessary, this can use `false, parallel, 1` instead.